packages/cli/src/commands/join.ts

```diff
@@ -174,21 +194,64 @@ packageVersion: string
         tag['x-displayName'] = tag['x-displayName'] || tag.name;
         tag.name = entrypointTagName;
         joinedDef.tags.push(tag);
+
+        if (skipTagsCheck) {
+          potentialConflicts.tags.description[entrypointTagName] = [entrypoint];
+        }
       }
-      if (!joinedDef[xTagGroups][indexGroup][Tags].find((t: any) => t === entrypointTagName)) {
-        joinedDef[xTagGroups][indexGroup][Tags].push(entrypointTagName);
+      // If tag already exist do not overwvite tag proprs,
+      // instead check if description are different for potential conflicts warning
+      else if (skipTagsCheck && tag.hasOwnProperty('description')) {
+        const isTagDescriptionNotEqual = joinedDef.tags.find(
+          (item: any) => item.description !== tag.description,
+        );
+        potentialConflicts.tags.description[entrypointTagName] = [
+          ...(potentialConflicts.tags.description[entrypointTagName] || []),
+          ...(isTagDescriptionNotEqual ? [entrypoint] : []),
+        ];
```

Comment:
Could it be like this?
```suggestion
        potentialConflicts.tags.description[entrypointTagName].push(...(isTagDescriptionNotEqual ? [entrypoint] : []);
```